instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/http/TemporaryMetricsView.java

```diff
@@ -42,34 +44,54 @@ private static Set<AttributeKey> buildDurationAlwaysInclude() {
     return view;
   }
 
-  private static Set<AttributeKey> buildDurationClientView() {
+  private static Set<AttributeKey> buildOldDurationAlwaysInclude() {
+    // the list of included metrics is from
+    // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#attributes
+    Set<AttributeKey> view = new HashSet<>();
+    view.add(SemanticAttributes.HTTP_METHOD);
+    view.add(SemanticAttributes.HTTP_STATUS_CODE); // Optional
+    view.add(SemanticAttributes.NET_PROTOCOL_NAME); // Optional
+    view.add(SemanticAttributes.NET_PROTOCOL_VERSION); // Optional
+    return view;
+  }
+
+  private static Set<AttributeKey> buildStableDurationClientView() {
     // We pull identifying attributes according to:
     // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#attribute-alternatives
-    // We only pull net.peer.name and net.peer.port because http.url has too high cardinality
-    Set<AttributeKey> view = new HashSet<>(durationAlwaysInclude);
-    view.add(SemanticAttributes.NET_PEER_NAME);
-    view.add(SemanticAttributes.NET_PEER_PORT);
-    view.add(SemanticAttributes.NET_SOCK_PEER_ADDR);
-    // stable semconv
+    Set<AttributeKey> view = new HashSet<>(stableDurationAlwaysInclude);
     view.add(NetworkAttributes.SERVER_SOCKET_ADDRESS);
     view.add(NetworkAttributes.SERVER_ADDRESS);
     view.add(NetworkAttributes.SERVER_PORT);
     return view;
   }
 
-  private static Set<AttributeKey> buildDurationServerView() {
+  private static Set<AttributeKey> buildOldDurationClientView() {
+    // We pull identifying attributes according to:
+    // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#attribute-alternatives
```

Comment:
This link does not point to the old attributes anymore (and we should probably point to `semantic-conventions` for the stable attribute set)